### ISSUE TITLE
Niagara Settings button

### ISSUE THREAD

**Issue by Ankoar** [8bitPit/Niagara-Issues#1595]
Dear Niagara-launcher team,

i've been using the Niagara launcher for a couple of months now and i'm generally satisfied.
There is just one little thing bothering me: The "Niagara Settings"-App button at the very bottom of the app drawer, leading you to the settings menu.

The reason why it's bothering me is, that its design absolutely doens't fit to my prefered Icon pack. I am aware that the shape of the button adapts to the overall app shape you choose, however i couldn't find any way to also change the texture/ icon image of that button.

All in all this is just a cosmetic issue, but maybe i could be solved quite fast, if you either would

> add the possibility to customize the button with for example the same way as you can already to it with other apps (hold the app button, tap the app symbol in the pop-up app menu)

or/ and

> give the option (in the settings) to deactivate / hide the "Niagara Settings"-Button in the app drawer. This also wouldn't affect the overall handleability of the launcher, since you can always access the Niagara settings from the pop-up-menu of any app.

I very much love using your launcher (my favourite launcher by far) and can say that the expiernce would get just about perfect if you could add in those small quality of life changes.

I'm curious about your feedback!
Keep up the great work!

Best regards
Nils

**Comment by Maxr1998** [comment]
Right, allowing custom icons is definitely planned, however, implementing it isn't a quick and easy task the way that part of our app is designed, so don't expect it very soon.

**Comment by rrigamondi** [comment]
On topic with this, would it be possible to at least select one of the preconfigured icons? I used to have google camera set for the niagara button and it'd show a cute camera icon, but since changing roms it shows a link icon instead. If there's automatic icon assignment it'd be neat to be able to pick manually between those already present

**Comment by Maxr1998** [comment]
We'll check if we can improve the automatic detection for such cases. I have a few ideas for that already.

**Comment by 8bitPit** [comment]
While we don't plan to provide individually changing the icons for the settings, pro, and recently installed buttons, we will allow third-party icon pack developers to provide custom icons.
